src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -133,6 +165,19 @@ permutation_matrix(R::NCRing, p::PermGroupElem) = permutation_matrix(R, Vector(p
 
 Return whether the matrix `B` is hermitian, i.e. `B = conjugate_transpose(B)`.
 Return `false` if `B` is not a square matrix, or the field has not even degree.
+
+# Examples
+```jldoctest
+julia> F, z = finite_field(2, 2); x = matrix(F, [1 z; 0 1]);
```

Comment:
You need either this
```suggestion
julia> F, z = finite_field(2, 2); x = matrix(F, [1 z; 0 1])
```